package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pino",
-  "version": "5.15.0",
+  "version": "5.16.0",
```

Comment:
We bump this on release, can you please revert the version change?

Reply:
Already did.